ops/terraform/remote_files/scripts/start-bacalhau.sh

```diff
@@ -97,5 +98,6 @@ else
     --web-ui-port 80 \
     --labels owner=bacalhau \
     --requester-job-translation-enabled \
-    --default-publisher ipfs
+    --default-publisher local
+    --local-publisher-address "${BACALHAU_NODE_COMPUTE_LOCAL_PUBLISHER_ADDRESS}"
```

Comment:
missing `\` after local. Also you don't have to set cli flag is you used the correct env variable, which is `BACALHAU_LOCAL_PUBLISHER_ADDRESS `

Reply:
I think this, and patience waiting for the build were the answer.